src/test/php/PDepend/Source/Language/PHP/Features/PHP80/MatchExpressionTest.php

```diff
@@ -58,14 +58,33 @@ class MatchExpressionTest extends PHPParserVersion80Test
      */
     public function testMatchExpression()
     {
+        $this->checkMatchExpression();
+    }
+
+    /**
+     * @return void
+     */
+    public function testMatchExpressionWithNamespace()
+    {
+        $this->checkMatchExpression('Baz');
+    }
+
+    /**
+     * @return void
+     */
+    private function checkMatchExpression(string $namespacePrefix = null)
+    {
+        $matchImage = implode('\\', array_filter([$namespacePrefix, 'match']));
```

Comment:
Fix PHP 5.3 compatibility
```suggestion
        $matchImage = implode('\\', array_filter(array($namespacePrefix, 'match')));
```